zabbix-probes/occi-zabbix-probe/src/main/java/org/indigo/occiprobe/openstack/CmdbFeignClient.java

```diff
@@ -18,5 +18,15 @@
 
   @RequestLine("GET /provider/id/{providerId}/has_many/services?include_docs=true")
   JsonElement providerInfo(@Param("providerId") String providerId);
+  
+  
+  @RequestLine("GET /provider/list")
+  JsonElement providerList();
+  
+  @RequestLine("GET /image/list")
+  JsonElement providerImages();
+  
+  @RequestLine("GET /provider/id/{providerId}/has_many/services?include_docs=true")
```

Comment:
I shouldn't had to touch occi-probe. Maybe I did while trying to move FeignClient to zabbix-probes-commons. Anyway, yes that's correct. I wrongly doubled it. just corrected